### PR TITLE
Add Matomo Tag Manager to core

### PR DESCRIPTION
**Pull request by tsteur**

### CODE REVIEW

**Comment by tsteur** [comment]
There's a failing system test in Anonymous tracking which will be fixed when merging (to avoid merging PR in submodules etc).

There are couple of failing UI tests but I believe some of them fail in 3.x-dev as well.

I've been trying to reproduce and fix this  failing screenshot https://builds-artifacts.matomo.org/matomo-org/matomo/tagmanager/29950/UIIntegrationTest_admin_manage_tracking_code.png but cannot reproduce it locally when running the UI tests and neither in the browser. Also not seeing any JS error in the UI build test logs.

**Comment by tsteur** [comment]
FYI: I'm currently waiting for a review of the wording on that page.

**Comment by tsteur** [comment]
Updated wording and added translations.

File: plugins/CorePluginsAdmin/Controller.php [hunk 1]
```diff
@@ -144,8 +146,52 @@ public function browseThemes()
...
+
+    public function disableActivateTagManagerPage()
+    {
+        $this->dieIfPluginsAdminIsDisabled();
+        Piwik::checkUserHasSomeAdminAccess();
+
+        $tagManagerTeaser = new TagManagerTeaser(Piwik::getCurrentUserLogin());
+
+        if (Piwik::hasUserSuperUserAccess()) {
+            $tagManagerTeaser->disableGlobally();
```
Comment (diosmosis) [inline plugins/CorePluginsAdmin/Controller.php]: Is it possible for a superuser to want to disable the tag manager for themselves only?

Comment (tsteur) [reply plugins/CorePluginsAdmin/Controller.php]: nope, cause then it wouldn't be possible anymore to disable it for all users.

Comment (diosmosis) [reply plugins/CorePluginsAdmin/Controller.php]: what i mean is, if one superuser might want to disable it for all users, while another might want to just disable it for themselves. ie, should the feature to disable all be separate from the feature to disable for the current logged in user (super user or not)

Comment (diosmosis) [reply plugins/CorePluginsAdmin/Controller.php]: Gotcha

**Comment by diosmosis** [comment]
screenshots need to be updated, otherwise lgtm (tested locally and it works)

**Pull request merged**